src/page.cpp

```diff
@@ -310,6 +310,13 @@ void Page::ResetAligners()
     AlignVerticallyFunctor alignVertically(doc);
     this->Process(alignVertically);
 
+    if (doc->IsNeumeLines()) {
+        AdjustXRelForTranscriptionFunctor adjustXRelForTranscription;
+        this->Process(adjustXRelForTranscription);
+        AdjustYRelForTranscriptionFunctor adjustYRelForTranscription;
+        this->Process(adjustYRelForTranscription);
+    }
```

Comment:
This this is transcription (facsimile) related, it is called from `Page::LayOutTranscription`. Could you explain why this needs to be added here?

Reply:
I need to call `Page::ResetAligners()` after dragging a staff to realign the staff children. The horizontal/vertical alignment is reset for accid so I need this to adjust `X/YRel` for accids so that it wont jump.

Reply:
Could you explain why accids behave any differently than neumes or ncs ? They should not

Reply:
Accids don't have a pitch, so they are skipped in `CalcAlignmentPitchPosFunctor`. After reset alignment functors `Page::ResetAligners()`, `X/YRel` of accids get reset to 0, I need to adjust them back

Reply:
hmmm I guess I should call `Page::LayOutTranscription` instead of `Page::ResetAligners`?

Reply:
Why not calling `Page::LayOutTranscription` instead of `Page::ResetAligners`? That would apply `AdjustYRelForTranscriptionFunctor`, wouldn't it?